### ISSUE TITLE
klix.ba tells me to stop using ad blocking software even with all lists enabled in ublock

### ISSUE THREAD

**Issue by asocial-creep** [uBlockOrigin/uAssets#7496]
### URL(s) where the issue occurs

`https://www.klix.ba/komentar/17188984`

### Describe the issue

When I enable all except other languages lists in ublock origin then I see the following message on that website: "You are seeing this message because ad or script blocking software is interfering with this page. Disable any ad or script blocking software, then reload this page."
That happens even if I try to use umatrix and ghostery with all lists enabled or disabled, or that two additional extensions enabled or disabled. I think that message did not appear when I did not use all lists in ublock, but only few.

### Screenshot(s)

![Untitled](https://user-images.githubusercontent.com/40727207/83472051-99bb8e00-a486-11ea-9423-fd4b8f4b49aa.jpg)

### Versions

- Browser/version: Google Chrome Version 83.0.4103.61 (Official Build) (64-bit)
- uBlock Origin version: 1.27.10

### Settings

- just enabled all lists except language ones

### Notes

I am using zenmate vpn with 1 week trial with currently brasil location, although I think it happens on any ip. Please don't tell me to hide that element because it has random id, currently it is "##.s3ch0p8h49jc" when picked via ublock. Should I use other trick or extension or will you fix it? Actually when I disabled EasyPrivacy and 2 tracking lists, and refreshed page the message is gone now, even if I load in incognito, although in normal browser I am loggen into klix.ba account. Could there something been loaded into cache in the meantime (when I realoaded with 3 lists disabled) and browser shares cache between ram/disk/incognito/normal? Can't make problem reappear now, although it reappeared 10 times before I disabled and reenabled 2 tracking lists (forgot which now) and easyprivacy.

**Comment by uBlock-user** [comment]
Test on default lists only, I can't reproduce. A filter already was added in uBlock Filters.

**Comment by asocial-creep** [comment]
I can't reproduce it either. Probably some smart trick to avoid detection used by admins of websites. And it never appears with default filter lists, but only sometimes appears with all lists.

**Comment by asocial-creep** [comment]
I can reproduce it again, but randomly, to be more specific, now that it appeared again here is the current state of extensions. All except languages lists are enabled in ublock and updated. uMatrix has blocked 3 domains and says the same on toolbar icon: certify-js.alexametrics.com, script.dotmetrics.net, securepubads.g.doubleclick.net. Ghostery also 3: Doubleclick, Google Tag Manager, Facebook Connect. uBlock says 14. In Ghostery I disabled any additional features like 3 badges for: enhanced ad blocking, enhanced anti tracking, smart blocking.  That message rectangle selected via chrome inspect element and copied:

`<div class="nodv7n7aofn4" style="bottom: 0px; left: 0px; position: fixed; width: 101%; z-index: 2147483560; background-color: rgb(250, 244, 220); box-shadow: rgb(136, 136, 136) 0px 0px 12px; color: rgb(2, 9, 2); display: flex; justify-content: center; font-family: Roboto, Arial;"><div class="jufn5vlxrxuu"></div><div class="d8zqnbwkcr4y"></div><div class="r56bofjzxjar"></div><div class="qaybknske84e" style="width: 81%; max-width: 750px; margin: 24px; display: flex; align-items: flex-start; justify-content: center;"><div class="j81qt8k09utp"></div><div class="ad1l15ygy8vf"></div><div class="uvtrphtmbfsq"></div><div class="ryj19n1up9w8"></div><img class="uty8oy1myqnz" src="https://www.gstatic.com/images/icons/material/system/1x/warning_amber_24dp.png" style="height: 24px; width: 24px; padding-right: 16px;"><div class="khxarnt4vyyr"></div><div class="dmts5m22gdmf"></div><div class="6qaek51klawp"></div><div class="j6vz2w90f2iw"></div><div class="4mby82fr45sp"></div><div class="ggj2kvkmy95m"></div><div class="1pi6nkb52iov"><div class="m0y3d9bakrhg"></div><div class="ensor3enwffc"></div><div class="e8zi6rdc9gda" style="font-weight: bold;">You are seeing this message because ad or script blocking software is interfering with this page.</div><div class="4mcb3vpxev8h"></div><div class="kd7oioh60mje"></div><div class="897dq3uop4no"></div><div class="ik57jv2cr9xb"></div><div class="kfykfjeiotcy"></div><div class="7m810k2rl0ck"></div><div class="ne6an8hsvb76"></div><div class="4l5bymokczaw">Disable any ad or script blocking software, then reload this page.</div><div class="prr264chqvxr"></div><div class="p4awfzek0ma6"></div></div><div class="4pbyq9yhvstw"></div><div class="8bsvf0oyl6im"></div><div class="uwmzlkn8tgg"></div><div class="t8kjxdxnrdrh"></div></div><div class="3qyzgr955uhc"></div><div class="3z33cfff9188"></div></div>`

Here is list of blocked elements reported by ublock's network monitor (logger):

`+12
||fundingchoicesmessages.google.com^$3p
--
www.klix.ba
3
xhr
https://fundingchoicesmessages.google.com/l/AGSKWxWcP1nFsod4qhSoDFojDCpowNMh1Uq2PlZSFGZg4RnNor6lD_n4ke_uGez5Pw2hioRKTkCDUjlxa1TEILPB?ab=1

+12
||connect.facebook.net^$3p,domain=~facebook.com|~fb.com
--
www.klix.ba
3
script
https://connect.facebook.net/en_US/sdk.js

+11
||fundingchoicesmessages.google.com^$3p
--
www.klix.ba
3
script
https://fundingchoicesmessages.google.com/f/AGSKWxUdKceaAPXs19zMtEreroD-kgV_AcK3N9znXJ1oTsb8WeLPZ1TQKcaqUK7Vju-FGcBSNqXHi1eiuDW1NybzlHI=

+11
||fundingchoicesmessages.google.com^$3p
--
www.klix.ba
3
script
https://fundingchoicesmessages.google.com/f/AGSKWxUdKceaAPXs19zMtEreroD-kgV_AcK3N9znXJ1oTsb8WeLPZ1TQKcaqUK7Vju-FGcBSNqXHi1eiuDW1NybzlHI=

+11
/icon_linkedin.
--
www.klix.ba
1
image
https://static.klix.ba/sc_icons/icon_linkedin.png

+10
||fundingchoicesmessages.google.com^$3p
--
www.klix.ba
3
script
https://fundingchoicesmessages.google.com/f/AGSKWxUdKceaAPXs19zMtEreroD-kgV_AcK3N9znXJ1oTsb8WeLPZ1TQKcaqUK7Vju-FGcBSNqXHi1eiuDW1NybzlHI=

+10
||alexametrics.com^
--
www.klix.ba
3
script
https://certify-js.alexametrics.com/atrk.js

+10
||connect.facebook.net^$3p,domain=~facebook.com|~fb.com
--
www.klix.ba
3
script
https://connect.facebook.net/en_US/fbevents.js

+10
||script.dotmetrics.net^
--
www.klix.ba
3
script
https://script.dotmetrics.net/door.js?id=1289

+10
||fundingchoicesmessages.google.com^$3p
--
www.klix.ba
3
script
https://fundingchoicesmessages.google.com/f/AGSKWxUdKceaAPXs19zMtEreroD-kgV_AcK3N9znXJ1oTsb8WeLPZ1TQKcaqUK7Vju-FGcBSNqXHi1eiuDW1NybzlHI=

+10
||connectad.io^$3p
--
www.klix.ba
3
script
https://cdn.connectad.io/connectmyusers.js

+10
/prebid2.
--
www.klix.ba
1
script
https://www.klix.ba/scripts/prebid2.44.7.js?v=6

+10
||doubleclick.net^
--
www.klix.ba
3
script
https://securepubads.g.doubleclick.net/tag/js/gpt.js

+10
||contributor.google.com^*/loader.js$3p
--
www.klix.ba
3
script
https://contributor.google.com/scripts/89ba68056c3e1aac/loader.js

+9
no-cosmetic-filtering: www.klix.ba true
--
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053

+0
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053
`

Oops, I realized it, when this exactly happens, although even then more or less randomly. When I disabled cosmetic filtering that message is gone now. Looks like it somehow detects that some elements are hidden by ublock. The following were reported in ublock log when I still did not disable cosmetic filters for the site:

`+19
##div[id^="div-gpt-ad"]
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053

+19
##.shareovi
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053

+19
##.shareovanje
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053

+19
##.sharebtns
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053

+19
##.icon_facebook
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053

+19
##.brojShareova
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053

+19
##.backtotop
www.klix.ba
dom
https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053`

And remember, this happened when I was logged in via facebook, maybe had to disable some filters just to be able to login earlier then reenable them.

Confirmed! As soon as I reenabled cosmetic filters the message is back!

Now on this page tested, although I think it appears on any klix.ba page: `https://www.klix.ba/vijesti/bih/u-federaciji-bih-tri-osobe-pozitivne-na-koronavirus-testirano-419-uzoraka/200602053`

**Comment by llacb47** [comment]
Are you sure your filters are up to date?

**Comment by asocial-creep** [comment]
> Are you sure your filters are up to date?

Yes, updated taday. As I said, only when I disable cosmetic filters then that warning disappears. Or when I disable some filter lists. Now when I have enabled only all ublock (internal) lists, Fanboy's Enhanced Tracking List, Fanboy's Annoyance List, EasyPrivacy, EasyList, problem is gone. Even if I keep cosmetic filtering active.

**Comment by asocial-creep** [comment]
Still difficult to hunt down. Now I used same case: all lists except foreign languages enabled and cosmetic enabled as always. But when I cleared cache in chrome of all time, problem disappeared. In incognito mode it never appeared meaning you had to be logged in to website via facebook or google. But now even when I am logged it does not appear anymore after clearing browser cache. With or without umatrix and ghostery.
